app/views/ParentsViewComponent.vue

```diff
@@ -0,0 +1,244 @@
+        p
+          span.spr(data-i18n="parent_landing.dive_par1")
+          a(href="https://www.learntechlib.org/p/19982/", data-i18n="parent_landing.dive_link1")
+          span.spl.spr(data-i18n="parent_landing.dive_par2")
+          a(href="http://webcache.googleusercontent.com/search?q=cache:v-UZ4U5_b7cJ:www.ejmste.com/A-Study-of-the-Effects-of-Digital-Learning-on-Learning-Motivation-and-Learning-Outcome,69635,0,2.html+&amp;cd=1&amp;hl=en&amp;ct=clnk&amp;gl=us", data-i18n="parent_landing.dive_link2")
+          span.spl.spr(data-i18n="parent_landing.dive_par3")
+          a(href="https://www.healio.com/nursing/journals/jcen/2016-8-47-8/%7B4c67edb9-1a9c-432d-af17-0f2856f3ce2c%7D/importance-of-gamification-in-increasing-learning", data-i18n="parent_landing.dive_link3")
+          span.spr(data-i18n="parent_landing.dive_par4")
+          a(href="http://www.sun.ac.za/english/learning-teaching/ctl/Documents/Gamification%20in%20education.pdf", data-i18n="parent_landing.dive_link4")
+          span.spl(data-i18n="parent_landing.dive_par5")
+        p
+          span.spr(data-i18n="parent_landing.dive_par6")
+          a(href="https://mitpress.mit.edu/books/ecology-games", data-i18n="parent_landing.dive_link5")
+          span.spr(data-i18n="parent_landing.dive_par7")
+          a(href="https://link.springer.com/chapter/10.1007/978-3-319-26508-7_20", data-i18n="parent_landing.dive_link6") 
+          span(data-i18n="parent_landing.dive_par8")
+    .row
+      #dive-graphic-3.col-lg-5.col-md-12.col-sm-12.text-center
+        div
+        h2(data-i18n="parent_landing.dive_head3")
+      .col-lg-6.col-md-12.col-sm-12.col-sm-offset-1
+        p
+          a(href="https://www.theatlantic.com/technology/archive/2012/05/six-reasons-non-computer-nerd-might-want-learn-code/328102/", data-i18n="parent_landing.dive_3_link1")
+          span.spl(data-i18n="parent_landing.dive_3_paragraph")
+    .row.text-center
+      .refer-btn
+        button.btn.btn-forest.btn-lg(v-on:click="$emit('referTeacher')", data-i18n="parent_landing.refer_teacher")
+    .row.focus-quote
+      h3.text-center(data-i18n="parent_landing.mission")
+    
+    #mission.row
+      .col-md-4.col-sm-12
+        img.img-responsive(src="/images/pages/parents/mission1.png")
+        p
+          b(data-i18n="parent_landing.mission1_heading")
+        p(data-i18n="parent_landing.mission1_copy")
+      .col-md-4.col-sm-12
+        img.img-responsive(src="/images/pages/parents/mission2.png")
+        p
+          b(data-i18n="parent_landing.mission2_heading")
+        p(data-i18n="parent_landing.mission2_copy")
+      .col-md-4.col-sm-12
+        img.img-responsive(src="/images/pages/parents/mission3.png")
+        p
+          b(data-i18n="parent_landing.mission3_heading")
+        p(data-i18n="parent_landing.mission3_copy")
+
+</template>
+
+<script lang="coffee">
+module.exports = Vue.extend({
+})
+</script>
+
+<style lang="sass">
+@import "../styles/style-flat-variables"
+
+$red: #7D0101
```

Comment:
Will this go into the global styles or only be applied to this view?

For example, I see this Vue component restricts itself:
https://github.com/codecombat/codecombat/blob/dc99e76/app/views/admin/OutcomesReportComponent.vue#L402

Reply:
Great pickup